packages/react/src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -30,13 +30,24 @@ export default class DatePickerInput extends Component {
      * control
      */
     labelText: PropTypes.node.isRequired,
+
+    /**
+     * Provide text to be read to screen readers so we can tell user that
+     * he/she can start using calendar dropdown with down arrow key
+     */
+    calendarDropdownAssistiveText: PropTypes.string,
   };
 
   static defaultProps = {
     pattern: '\\d{1,2}\\/\\d{1,2}\\/\\d{4}',
     type: 'text',
     disabled: false,
     invalid: false,
+    calendarDropdownAssistiveText: [
+      'Use down arrow key to navigate to calendar dropdown.',
+      'Use arrow keys to navigate through dates.',
+      'Use control-arrow keys to increment and decrement year and month.',
+    ].join(' '),
```

Comment:
just wondering, is there a reason for this being a joined array?

Reply:
No reason, just from my old muscle memory of when string concatenation was slow (no longer the case because many modern JavaScript engine has lazy-evaluation of concatenated strings).